src/lib.rs

```diff
@@ -85,24 +85,14 @@
 
 #[macro_use]
 extern crate error_chain;
-extern crate handlebars;
-extern crate itertools;
 #[macro_use]
 extern crate lazy_static;
 #[macro_use]
 extern crate log;
-extern crate memchr;
-extern crate pulldown_cmark;
-extern crate regex;
-extern crate serde;
 #[macro_use]
 extern crate serde_derive;
```

Comment:
you can use macro imports and get rid of extern crates for these as well

Reply:
Thanks. But I think we should land it as is. Because we already transit
new edition. Also because I don't want to deal with missing macro importing
somewhere while trying to get rid of those `extern crate`.

Reply:
cool no issues